kolibri/plugins/coach/assets/src/app.js

```diff
@@ -196,10 +196,10 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.EXERCISE_RENDER,
```

Comment:
I know you didn't do this, but could we change this to a more semantic name like `EXERCISE_DETAIL_REPORT`?